docs/framework/TOC.md

```diff
@@ -1,4 +1,18 @@
 # [.NET Framework Development Guide](development-guide.md)
-# [Developing Web Applications with ASP.NET](develop-web-apps-with-aspnet.md)
-# [Developing Client Applications with the .NET Framework](develop-client-apps.md)
-# [64-bit Applications](64-bit-apps.md)
+## [Application essentials](../standard/application-essentials.md)
+## [Data and modeling](data/index.md)
+## [Client applications](develop-client-apps.md)
+## [Web applications with ASP.NET](develop-web-apps-with-aspnet.md)
+## [Service-oriented applications with WCF](windows-services/index.md)
+## [Building workflows with Windows Workflow Foundation](windows-workflow-foundation/index.md)
+## [Parallel processing and concurrency](../standard/parallel-processing-and-concurrency.md)
+## [Network programming in the .NET Framework](network-programming/index.md)
```

Comment:
This one uses the doc title, but most of these (e.g., *Application essentials*, *Data and modeling*, *Client applications*, etc.) are abbreviated titles. Make this *Network programming* to match?